src/main/webapp/app/exam/participate/summary/exam-participation-summary.component.ts

```diff
@@ -90,6 +90,18 @@ export class ExamParticipationSummaryComponent implements OnInit {
         }
     }
 
+    /**
+     * A guard function used for the ComplaintInteractionsComponent to indicate whether complaint submissions are valid.
+     * These are only allowed if they are submitted within the student review period.
+     */
+    isWithinStudentReviewPeriod(): boolean {
+        if (this.studentExam.exam.examStudentReviewStart && this.studentExam.exam.examStudentReviewEnd) {
+            const present = moment();
```

Comment:
maybe we should use the server time instead - otherwise students might can cheat the review times on the client. I don't know if that can be a problem, because we have the check on the server, but might be a problem if a student is in a different time zone? Not sure though

Reply:
Yes that is a good idea, i will use the server date